examples/iaas/iaas.go

```diff
@@ -130,7 +130,7 @@ func main() {
 	fmt.Printf("[Iaas API] Current state of the network: %q\n", *network.State)
 	fmt.Println("[Iaas API] Waiting for network to be created...")
 
-	network, err = wait.CreateNetworkWaitHandler(context.Background(), iaasClient, projectId, httpResp.Header.Get("x-request-id")).WaitWithContext(context.Background())
```

Comment:
We can now remove the `WithCaptureHTTPResponse` from the call above

Reply:
removed